### DISCUSSION TITLE
Cannot compile / deploy my cart transform extension

### DISCUSSION THREAD

**Discussion by dekstech** [Shopify/function-examples#259] [Feedback: Cart Transform API]
I keep getting the following error:

 cart-merge-expand │   Optimizing with wasm-opt
 cart-merge-expand │ [wasm-validator error in function 1] unexpected false: all used features should be allowed, on

and it gives this error over and over again...any idea what this could be?

**Comment by adampetro** [comment]
Resolved in #255
